packages/members-api/lib/repositories/product.js

```diff
@@ -133,14 +134,15 @@ class ProductRepository {
         }
 
         const productData = {
+            type: 'paid',
```

Comment:
Fixes type as `paid` for any new tiers that are created, ignoring any other value coming in, since we only allow `paid` tiers to be created and `free` is inserted via fixture.